### PR TITLE
add "crate_version" variable to template

### PR DESCRIPTION
**Pull request by LuckyTurtleDev**

### CODE REVIEW

**Comment by LuckyTurtleDev** [comment]
ci does also fail on `main` see:
https://github.com/msrd0/cargo-doc2readme/actions/runs/5413828614/jobs/9839974194

File: src/input.rs [hunk 1]
```diff
@@ -303,6 +303,8 @@ pub enum TargetType {
 pub struct InputFile {
 	/// The name of the crate.
 	pub crate_name: String,
+	/// The version of the crate
+	pub crate_version: String,
```
Comment (msrd0) [inline src/input.rs:307]: Why do we need this? The version of the crate should already be in the dependencies array

Comment (LuckyTurtleDev) [reply src/input.rs:307]: Yes, but this does required the name of your crate. If you do rename your crate, you have to manual change all access to the array.
I think it is a good option to get the version of your crate by `crate_version`. Similar to `crate`, whitch provides the crate name.

Comment (msrd0) [reply src/input.rs:307]: well at least we shouldn't convert it to a `String` here

Comment (msrd0) [reply src/input.rs:307]: I don't know, but this struct is not the input for minijinja

Comment (LuckyTurtleDev) [reply src/input.rs]: done

**Pull request merged**